docs/_data/meetups/usa-austin.yaml

```diff
@@ -7,9 +7,8 @@ meetup: WriteTheDocs-ATX-Meetup
 organizers:
   - name: Shrie Spangler
     link: https://twitter.com/shrielenee
-  - name: Becky Todd
-    link: https://twitter.com/beckatodd
   - name: Margaret Eker
     link: https://twitter.com/meker
   - name: Mary Connor
     link: https://twitter.com/maryfconnor
+  - name: Rick McDonald
```

Comment:
This needs a link as well I think. 
You can use the meetup link if Rick doesn't have a site or twitter account.

Reply:
Gotcha!